.travis.yml

```diff
@@ -140,8 +142,9 @@ jobs:
             - libstdc++-5-dev
             - libubsan0
             - parallel
+            - libc6-dev-i386
       before_install:
-        - mkdir bin ; ln -s /usr/bin/clang-3.7 bin/gcc
+        - mkdir bin ; ln -s /usr/bin/gcc-5 bin/gcc
```

Comment:
I don't understand this change? Is `gcc-5` even installed in this configuration?

Reply:
The problem is that clang on Travis won't allow the -m32.

Reply:
That seems rather weird - what's the error message?

Reply:
It can't find the 32-bit headers; clang has it's own headers, so there may well be a package that adds the 32-bit headers for clang.

Reply:
Ok, it still seems a bit magical that `gcc-5` is even present. But ok, it works... As this might break at some random future point (or, equally, clang packages be fixed), could the commit message please be made verbose to explain what is going on here? Moving to some other version of GCC will be painful if it cannot be found out what is going on in those lines.

Reply:
> it still seems a bit magical that gcc-5 is even present. 

`libstdc++-5-dev` (https://github.com/diffblue/cbmc/pull/2217/files#diff-354f30a63fb0907d4ad57269548329e3R169) depends on package `gcc-5`